### ISSUE TITLE
+ symbols in email addresses not parsing correctly and converting to @ symbols

### ISSUE THREAD

**Issue by wakatara** [marcwebbie/passpie#103]
Strange little one. If you type a + into an email address it gets converted into an @ sign in the following example (which is quite common for using gmail type aliases for accounts.):

*Steps to Reproduce*

If you type this in the command line  (I have ppa aliased to passpie add)

```
ppa singapore-accounts+domain@companydomain.io@companydomain/domain
```

*Observed Behaviour*

Your will end up with the following (which is wrong)

```
Name: companydomain.io@companydomain/domain
Login: singapore-accounts@domain
```

*Expected Behaviour*

Should see 

```
Name: companydomain/domain
Login: singapore-accounts+domain@companydomain.io
```

Lovig passpie btw, though the update command not working and this are a bit of a bummer. Still, awesome little app Thanks for all your hard work!

Daryl.

**Comment by marcwebbie** [comment]
Hello @wakatara,

I did ended up with the expected name/login:

![screen shot 2016-05-22 at 1 22 50 am](https://cloud.githubusercontent.com/assets/2798771/15452167/d2d7cfea-1fbb-11e6-80ae-07ff17cdd403.png)

+ What version of Passpie you have installed?
+ And what happens if you add the credential with quotes? 
```
ppa "singapore-accounts+domain@companydomain.io@companydomain/domain"
```

Thanks for your contribution. 😃

**Comment by wakatara** [comment]
Hey Marc!

```
🚀  ~ pp --version
passpie, version 1.4.1
🚀  ~ python --version
Python 2.7.11
🚀  ~ python3 --version
Python 3.5.1
```
on OSX El Capitan 10.11.5 (latest update)
I'm using the standard OSX Terminal with tmux 2.1

Passpie, tmux, and python all installed via homebrew.

It definitely works with the quotation marks (and, er... strangely seems to work without them now, making me think I might have double fat-fingered the entry when I tested.).

Thanks for fixing the passpie update "bug" though. That'll be a nice fix if that was the problem

And thanks for all your work on this! (I've converted 3 people in my office over to using passpie already... =] ).
